lib/modules/datasource/maven/util.ts

```diff
@@ -7,8 +9,8 @@ import { ExternalHostError } from '../../../types/errors/external-host-error';
 import type { Http } from '../../../util/http';
 import type { HttpResponse } from '../../../util/http/types';
 import { regEx } from '../../../util/regex';
-import { normalizeDate } from '../metadata';
 
+import { normalizeDate } from '../metadata';
```

Comment:
why this change?

Reply:
I don't know, I'm not even sure what changed there. It probably came from my IDE or linting.